ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialLiveShareLayout.kt

```diff
@@ -0,0 +1,157 @@
+@file:Suppress("UNUSED_EXPRESSION")
+
+package org.cru.godtools.tutorial.layout
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.LottieConstants
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+import org.ccci.gto.android.common.androidx.compose.foundation.text.minLinesHeight
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import org.cru.godtools.tutorial.Page
+import org.cru.godtools.tutorial.R
+
+@Composable
+internal fun TutorialLiveShareLayout(
+    nextPage: () -> Unit = {},
+    onTutorialAction: (Int) -> Unit = {},
+    page: Page,
+    anim: Int? = null,
+    title: Int,
+    body: Int,
+    img: Int? = null
+) = GodToolsTheme() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(
+                top = dimensionResource(R.dimen.tutorial_page_inset_top),
+                bottom = dimensionResource(R.dimen.tutorial_page_inset_bottom)
+            ).verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.minLinesHeight(5, MaterialTheme.typography.bodyMedium).padding(top = 16.dp))
```

Comment:
you need to define the padding before you set the height of the Spacer. the order of modifiers matters. The order flows from the outer layer to the inner layer. (see: https://stackoverflow.com/a/64216931/4721910 for an example of how order matters)

so in this case you are currently setting the height and then the padding. This causes the padding to be taken out of the allocated height. What you want to do is specify the padding before you specify the height of the Spacer.

for reference: 
`Modifier.size(40.dp).padding(8.dp)` would give you a 40dp square element with 8dp of padding inside the 40dp square.
`Modifier.padding(8.dp).size(40.dp)` would give you 8dp of padding around a 40dp square element